TexasHoldem.c

```diff
@@ -152,27 +152,53 @@ int playerChoice(void)
 //i made this function for the bets, but I don't know how the blinds and different roles factor in
 void DefineBet(int playerNum)
 {
+	double userbet;
+	//Start with first player as dealer, players are 1, 2, 3, and 4, going clockwise
+	int dealer = 1;
+
+	//Define initial bets
 	switch(playerNum)
 	{
 		//one player 3 cpu
 		case 1:
-		printf("What is Player 1's bet? ");
+		printf("Player %d is the dealer, player %d is the small blind, and player %d is the big blind", dealer, (dealer+1)%4, (dealer+2)%4;
+		//cpus are players 1, 2, and 3
+		if ((dealer+3)%4 == 4)
+		{
+			printf("What is Player %d's bet? ", (dealer+3)%4;
+			scanf("%lf", playerbets[(dealer+3)%4]);
```

Comment:
don't forget & for the scanf

Reply:
this happens three more times in the function, so I'm guessing you juts copy pasted it lol. so take a look at the scanf's in this